### ISSUE TITLE
Add Content under Step 2 Vagrant Tutorial Troubleshooting Section

### ISSUE THREAD

**Issue by YangLou1** [open-learning-exchange/open-learning-exchange.github.io#1525]
I run the Windows 8.1 and Above script to install all BeLL's Dependencies including VirtualBox and Vagrant. 
I suspended my virtual machine using the command vagrant suspend. Then I issued the command vagrant up in order to restart my machine. However, I got an error message saying "Vagrant up: There was an error while executing `VBoxManage`, a CLI used by Vagrant". My OS is Windows 8.1.

**Comment by vishwakarmaanil** [comment]
Re installing other version of virtual box ( 5.1.x ) solves this problem. Don’t install 5.2 version of virtual box  since it is not supported by vagrant installations.

**Comment by YangLou1** [comment]
I found the same error that someone was facing and its possible solutions at this link: https://github.com/hashicorp/vagrant/issues/1809. 
I fixed the error by killing the process "VBoxHeadless" in Task Manager (Windows 8.1), and the command vagrant up worked afterwards.

We can add the following content under Step 2 Vagrant Tutorial Troubleshooting Section as the 4th item:
When you run vagrant up from command prompt on windows after running vagrant suspend, you might get the following error: 
"Vagrant up: There was an error while executing VBoxManage, a CLI used by Vagrant". A possible solution is killing the process "VBoxHeadless" in Task Manager, and the command vagrant up might work afterwards. For more information visit [here](https://github.com/hashicorp/vagrant/issues/1809)
